new/release-cadence.md

```diff
@@ -8,14 +8,34 @@ Type: Informational
 
 # Chef Release Cadence
 
-Chef is released once per month, in the first week of the month unless delayed
-by the Chef Software Release Engineering team.
+Chef follows [Semantic Versioning](https://semver.org/) for releases. Major
+versions (eg. 11.x -> 12.x) will include backwards-incompatible changes, minor
+versions (eg 12.1 -> 12.2) will include new features and bug fixes but will be
+backwards-compatible to the best of our ability. Patch versions are governed
+by [RFC 47](rfc047-release-process.md).
+
+Chef feature releases are promoted to the stable channel once per month, in the
+second week of the month unless delayed by the Chef Software Release Engineering
```

Comment:
Ah, I meant that I don't think we need to talk about RelEng. Also I would like to eventually get at least the concept of any maintainer doing releases…

```
Chef feature releases are promoted to the stable channel once per month. It is expected that this occur during the second week of the month unless circumstances intervene.
```
Something like that?